packages/yoastseo/spec/scoring/InclusiveLanguageAssessorSpec.js

```diff
@@ -0,0 +1,214 @@
+import Paper from "../../src/values/Paper";
+import InclusiveLanguageAssessor from "../../src/scoring/inclusiveLanguageAssessor";
+import EnglishResearcher from "../../src/languageProcessing/languages/en/Researcher";
+
+// Note that the order of the testing suites is important because, "Jest executes all describe handlers before it executes any of the actual tests."
+// Source: https://jestjs.io/docs/setup-teardown
+
+describe( "a test for for the inclusive language assessor when run without its second parameter, options", () => {
+	it( "tests whether the assessor will be run without throwing an error using a ses assessment as an example", () => {
+		const mockPaper = new Paper( "This is a test with the non-inclusive phrase illegal immigrants." );
+		const researcher = new EnglishResearcher( mockPaper );
+		const assessor = new InclusiveLanguageAssessor( researcher );
+		assessor._assessments.forEach( assessment => {
+			if ( assessment.isApplicable( mockPaper, researcher ) ) {
+				expect( assessment.category ).toEqual( "ses" );
+				expect( assessment.learnMoreUrl ).toEqual( "<a href='https://yoa.st/inclusive-language-ses' target='_blank'>" );
+			}
+		} );
+	} );
+} );
+
+describe( "a test for for the inclusive language assessor when run in WordPress", () => {
+	const options = {
+		infoLinks: {},
+	};
+	it( "tests the category of the age assessments and the correct WP link", () => {
+		const mockPaper = new Paper( "This is a test with the non-inclusive word elderly." );
+		const researcher = new EnglishResearcher( mockPaper );
+		const assessor = new InclusiveLanguageAssessor( researcher, options );
+		assessor._assessments.forEach( assessment => {
+			if ( assessment.isApplicable( mockPaper, researcher ) ) {
+				expect( assessment.category ).toEqual( "age" );
+				expect( assessment.learnMoreUrl ).toEqual( "<a href='https://yoa.st/inclusive-language-age' target='_blank'>" );
+			}
+		} );
+	} );
```

Comment:
I would test this differently.

Right now this tests:
* whether the "elderly" assessment is applicable on a paper with the text "This is a test with the non-inclusive word elderly.".
* whether this specific assessment has, if it is applicable, the correct category and "learn more" URL.

I would test it by going through the list of assessments and checking if each assessment has the right learn more URL instead. Something like this:

```js
	const options = {
		infoLinks: {},
	};

	const expectedLinks = {
		age: "<a href='https://yoa.st/inclusive-language-age' target='_blank'>",
		appearance: "<a href='https://yoa.st/inclusive-language-appearance' target='_blank'>",
		ses: "<a href='https://yoa.st/inclusive-language-ses' target='_blank'>",
		gender: "<a href='https://yoa.st/inclusive-language-gender' target='_blank'>",
		other: "<a href='https://yoa.st/inclusive-language-other' target='_blank'>",
		disability: "<a href='https://yoa.st/inclusive-language-disability' target='_blank'>",
		culture: "<a href='https://yoa.st/inclusive-language-culture' target='_blank'>",
	};

	it( "tests that each assessment gets the correct WP link", () => {
		const mockPaper = new Paper( "This is a test with the non-inclusive word elderly." );
		const researcher = new EnglishResearcher( mockPaper );
		const assessor = new InclusiveLanguageAssessor( researcher, options );
		assessor._assessments.forEach( assessment => {
			expect( assessment.learnMoreUrl ).toEqual( expectedLinks[ assessment.category ] );
		} );
	} );
```

And add a separate test to check whether each assessment has the right category?

But let's not let this block merging this PR, since it is about the tests rather than the code 🙂 .

Reply:
What I want to say is that we want to test the end result of calling the Inclusive language assessor constructor: an Inclusive language assessor that has the correct list of assessments with the right properties.

Reply:
Good suggestion! 👍 I'll create a technical debt issue for this later today (or tomorrow). Let's try and pick it up for 19.13 😄